goodkey/weak.go

```diff
@@ -0,0 +1,62 @@
+package goodkey
+
+// This file defines a basic method for testing if a given RSA public key is on one of
+// the Debian weak key lists and is therefore considered easily enumerable. Instead of
```

Comment:
"...and the private key is therefore considered compromised" insead of "easily enumerable"